plugin/client_rpc.go

```diff
@@ -242,10 +242,10 @@ func (g *apiRPCClient) LoadPluginConfiguration(dest interface{}) error {
 	_args := &Z_LoadPluginConfigurationArgsArgs{}
 	_returns := &Z_LoadPluginConfigurationArgsReturns{}
 	if err := g.client.Call("Plugin.LoadPluginConfiguration", _args, _returns); err != nil {
-		log.Printf("RPC call to LoadPluginConfiguration API failed: %s", err.Error())
+		g.log.Error("RPC call to LoadPluginConfiguration API failed: ", mlog.Err(err))
```

Comment:
The `apiRPCClient` only has a `g.LogError` method.

`g.log` is only available for `hooksRPCClient`

Reply:
Done!